src/base/preferences.cpp

```diff
@@ -1459,10 +1468,9 @@ void Preferences::upgrade()
     QStringList labels = value("TransferListFilters/customLabels").toStringList();
     if (!labels.isEmpty()) {
         QVariantMap categories = value("BitTorrent/Session/Categories").toMap();
-        foreach (const QString &label, labels) {
+        foreach (const QString &label, labels)
```

Comment:
please revert this one, was better before IMO.

Reply:
This changes was made by uncrustify rules, for me is better to use `{` every time.

Reply:
done